packages/hardhat-zksync-vyper/test/fixture-projects/simple-docker-no-image/hardhat.config.ts

```diff
@@ -4,7 +4,7 @@ import { HardhatUserConfig } from 'hardhat/config';
 
 const config: HardhatUserConfig = {
     zkvyper: {
-        version: 'latest',
+        version: '1.5.7',
```

Comment:
Can we use 1.5.10 here similarly to how the latest zksolc 1.5.15 is used everywhere else?